packages/devtools_app/lib/src/standalone_ui/vs_code/flutter_panel.dart

```diff
@@ -17,7 +20,61 @@ import 'debug_sessions.dart';
 import 'devices.dart';
 import 'devtools/devtools_view.dart';
 
-/// A general Flutter sidebar panel for embedding inside IDEs.
+/// A general Flutter sidebar panel for embedding inside editors.
+///
+/// Provides some basic functionality to improve discoverability of features
+/// such as creation of new projects, device selection and DevTools features.
+class DtdEditorSidebarPanel extends StatefulWidget {
+  // TODO(dantup): Remove the Dtd prefix from these classes when the postMessage
+  //  versions are removed.
+  const DtdEditorSidebarPanel(
+    this.dtd, {
+    super.key,
+  });
+
+  final DartToolingDaemon dtd;
+
+  @override
+  State<DtdEditorSidebarPanel> createState() => _DtdEditorSidebarPanelState();
+}
+
+class _DtdEditorSidebarPanelState extends State<DtdEditorSidebarPanel> {
+  _DtdEditorSidebarPanelState();
+
+  Future<EditorClient>? _editor;
+
+  @override
+  void initState() {
+    super.initState();
+    // TODO(dantup): Should we make a new entry here for EditorSidebar so that
+    //  a) it's not VS Code-specific, and
+    //  b) we can track progress of the migration from postMessage?
```

Comment:
Correct. I presume you don't need to set up anything in analytics to handle the new value? I've used `editorSidebar` (and left the original one as `vsCodeFlutterSidebar`... I did initially rename it to `legacyVsCodeFlutterSidebar` but then realised it would fragment new stats from the previous ones so reverted that).

Reply:
Nope we will just start getting events with the new string. And yes, you are correct we should leave the legacy name as it was so we don't fragment the data